src/shared/lib/customTabs.ts

```diff
@@ -27,14 +28,13 @@ export function showCustomTab(div:HTMLDivElement, tab:ICustomTabConfiguration, u
 
     const runCallback = (tab:ICustomTabConfiguration)=>{
         if (getBrowserWindow()[tab.mountCallbackName]) {
-            getBrowserWindow()[tab.mountCallbackName](div, tab, url, store);
+            getBrowserWindow()[tab.mountCallbackName](div, tab, url, store, autorun);
         } else {
             alert(`Callback for tab ${tab.title} not found`);
         }
     }
 
     tab.dependencyPromise!.then(()=>{
-        alert("monkeys");
```

Comment:
🙈

Reply:
what? monkeys care about custom tabs and want to be kept informed